packages/back-end/types/organization.d.ts

```diff
@@ -68,6 +81,7 @@ export interface ExpandedMember extends Member {
   email: string;
   name: string;
   verified: boolean;
+  userPermissions?: UserPermissions;
```

Comment:
This is not needed anymore since the user's permissions are being sent down in a separate top-level field